### DISCUSSION TITLE
Recommended way to install for latest version on PopOS/Ubuntu?

### DISCUSSION THREAD

**Discussion by francislavoie** [kovidgoyal/kitty#7432] [General]
The package in the Ubuntu/PopOS repos is quite outdated. Is there some unofficial apt repo I can use to get the latest version, reliably? Or is there another recommended installation method to not fall behind?

**Comment by kovidgoyal** [comment]
https://sw.kovidgoyal.net/kitty/binary/

On Tue, May 14, 2024 at 04:35:22PM -0700, Francis Lavoie wrote:
> The package in the Ubuntu/PopOS repos is quite outdated. Is there some unofficial apt repo I can use to get the latest version, reliably? Or is there another recommended installation method to not fall behind?
>
> --
> Reply to this email directly or view it on GitHub:
> https://github.com/kovidgoyal/kitty/discussions/7432
> You are receiving this because you are subscribed to this thread.
>
> Message ID: ***@***.***>

--
_____________________________________

Dr. Kovid Goyal
https://www.kovidgoyal.net
https://calibre-ebook.com
_____________________________________

**Reply by francislavoie** [reply]
That makes sense. I made myself a little script in my `~`

```sh
#!/bin/bash

# https://sw.kovidgoyal.net/kitty/binary/

curl -L https://sw.kovidgoyal.net/kitty/installer.sh | sh /dev/stdin \
    launch=n
```

I was sorta hoping for something that updates with the rest of the system, but I can live with this.

**Comment by newptcai** [comment]
I have noticed that (at the moment of writing), on Ubuntu 24.04 you will get Kitty 0.32, which for whatever reason is very very slow on laptop. Using the binary installation, I have installed kitty 0.35.2, which is significantly faster.

Also, if you want to set kitty to default terminal emulator on Ubuntu after binary installation, you can use the method provided here: https://github.com/kovidgoyal/kitty/issues/1101#issuecomment-617100609
